src/PackageVersions/Versions.php

```diff
@@ -3,27 +3,80 @@
+  'phar-io/version' => '1.0.1@a70c0ced4be299a63d32fa96d9281d03e94041df',
+  'phpdocumentor/reflection-common' => '1.0.1@21bdeb5f65d7ebf9f43b1b25d404f87deab5bfb6',
+  'phpdocumentor/reflection-docblock' => '4.1.1@2d3d238c433cf69caeb4842e97a3223a116f94b2',
+  'phpdocumentor/type-resolver' => '0.4.0@9c977708995954784726e25d0cd1dddf4e65b0f7',
+  'phpspec/prophecy' => 'v1.7.2@c9b8c6088acd19d769d4cc0ffa60a9fe34344bd6',
+  'phpunit/php-code-coverage' => '5.2.3@8e1d2397d8adf59a3f12b2878a3aaa66d1ab189d',
+  'phpunit/php-file-iterator' => '1.4.2@3cc8f69b3028d0f96a9078e6295d86e9bf019be5',
+  'phpunit/php-text-template' => '1.2.1@31f8b717e51d9a2afca6c9f046f5d69fc27c8686',
+  'phpunit/php-timer' => '1.0.9@3dcf38ca72b158baf0bc245e9184d3fdffa9c46f',
+  'phpunit/php-token-stream' => '2.0.1@9a02332089ac48e704c70f6cefed30c224e3c0b0',
+  'phpunit/phpunit' => '6.4.4@562f7dc75d46510a4ed5d16189ae57fbe45a9932',
+  'phpunit/phpunit-mock-objects' => '4.0.4@2f789b59ab89669015ad984afa350c4ec577ade0',
+  'psr/log' => '1.0.2@4ebe3a8bf773a19edfe0a84b6585ba3d401b724d',
+  'sebastian/code-unit-reverse-lookup' => '1.0.1@4419fcdb5eabb9caa61a27c7a1db532a6b55dd18',
+  'sebastian/comparator' => '2.1.0@1174d9018191e93cb9d719edec01257fc05f8158',
+  'sebastian/diff' => '2.0.1@347c1d8b49c5c3ee30c7040ea6fc446790e6bddd',
+  'sebastian/environment' => '3.1.0@cd0871b3975fb7fc44d11314fd1ee20925fce4f5',
+  'sebastian/exporter' => '3.1.0@234199f4528de6d12aaa58b612e98f7d36adb937',
+  'sebastian/global-state' => '2.0.0@e8ba02eed7bbbb9e59e43dedd3dddeff4a56b0c4',
+  'sebastian/object-enumerator' => '3.0.3@7cfd9e65d11ffb5af41198476395774d4c8a84c5',
+  'sebastian/object-reflector' => '1.1.1@773f97c67f28de00d397be301821b06708fca0be',
+  'sebastian/recursion-context' => '3.0.0@5b0cd723502bac3b006cbf3dbf7a1e3fcefe4fa8',
+  'sebastian/resource-operations' => '1.0.0@ce990bb21759f94aeafd30209e8cfcdfa8bc3f52',
+  'sebastian/version' => '2.0.1@99732be0ddb3361e16ad77b68ba41efc8e979019',
+  'seld/cli-prompt' => '1.0.3@a19a7376a4689d4d94cab66ab4f3c816019ba8dd',
+  'seld/jsonlint' => '1.6.1@50d63f2858d87c4738d5b76a7dcbb99fa8cf7c77',
+  'seld/phar-utils' => '1.0.1@7009b5139491975ef6486545a39f3e6dad5ac30a',
+  'symfony/console' => 'v3.3.13@63cd7960a0a522c3537f6326706d7f3b8de65805',
+  'symfony/debug' => 'v3.3.13@74557880e2846b5c84029faa96b834da37e29810',
+  'symfony/event-dispatcher' => 'v3.3.13@271d8c27c3ec5ecee6e2ac06016232e249d638d9',
+  'symfony/filesystem' => 'v3.3.13@77db266766b54db3ee982fe51868328b887ce15c',
+  'symfony/finder' => 'v3.3.13@138af5ec075d4b1d1bd19de08c38a34bb2d7d880',
+  'symfony/polyfill-mbstring' => 'v1.6.0@2ec8b39c38cb16674bbf3fea2b6ce5bf117e1296',
+  'symfony/process' => 'v3.3.13@a56a3989fb762d7b19a0cf8e7693ee99a6ffb78d',
+  'theseer/tokenizer' => '1.1.0@cb2f008f3f05af2893a87208fe6a6c4985483f8b',
+  'webmozart/assert' => '1.2.0@2db61e59ff05fe5126d152bd0655c9ea113e550f',
+  'ocramius/package-versions' => '2.0.x-dev',
```

Comment:
Uhm... This doesn't look right to me. The hash should be there anyway.

Reply:
The hash is there, that output was due to a previous iteration during my development where operator precedence tricked me. In fact I hadn't touched any other test apart from adding mine, so nothing has changed in expected behavior.